etc/docker/elyra/Dockerfile.dev

```diff
@@ -15,28 +15,66 @@
 # limitations under the License.
 #
 
-# Ubuntu 18.04 LTS - Bionic
+# Ubuntu 20.04.2 LTS (Focal Fossa)
 # Repository: https://hub.docker.com/r/jupyterhub/k8s-singleuser-sample/tags
 FROM jupyterhub/k8s-singleuser-sample:1.2.0
 
+USER root
 
-ARG TAG="dev"
+# Install build dependencies
```

Comment:
Could we combine the runs below to make less layers

Reply:
how much time/space would this save? Looks like cleanup is being done at every step and assuming the os libs and requirement install layers arent updated would make iterative building quicker? and this makes things more readable?

Reply:
I think it's not necessary to over-optimize the number of layers (this is a dev file), and even if someone wanted to extend this image, there are over 100 layers remaining.

I agree with @akchinSTC